README.md

```diff
@@ -294,6 +294,7 @@ No Wayland-specific requirements, so you can use your xorg solution of choice to
 * [Wallutils](https://github.com/xyproto/wallutils) - A set of utilities to manage monitors, resolutions, wallpapers and timed wallpapers
 * [wpaperd](https://github.com/danyspin97/wpaperd) - Wallpaper daemon that shows random wallpapers from a directory and changes them after some time
 * [wbg](https://codeberg.org/dnkl/wbg) - Super simple wallpaper application for Wayland compositors
+* [waypaper](https://github.com/anufrievroman/waypaper) - GUI wallpaper setter for Wayland inspired by Nitrogen
```

Comment:
Since it requires [`swaybg` or `swww`](https://github.com/anufrievroman/waypaper/blob/eaa94ec4e4391103837f600ed3527c2a1d6bf3cd/waypaper/changer.py#L6) to work, would it perhaps make sense to slightly change the description to something like: "_GUI frontend for swaybg/swww to switch wallpaper_"?

Reply:
Sure, makes sense.